src/accelerate/utils/modeling.py

```diff
@@ -616,7 +616,73 @@ def check_tied_parameters_on_same_device(tied_params, device_map):
             )
 
 
-def find_tied_parameters(model: nn.Module, **kwargs):
+def _get_named_modules(
+    module: torch.nn.Module,
+    memo: Optional[Set[torch.nn.Module]] = None,
+    prefix: str = "",
+    remove_duplicate: bool = True,
+):
+    """
+    Return an iterator over all modules in the network, yielding both the name of the module as well as the module
+    itself. Copied from PyTorch `torch.nn.Module.named_modules` for compatability with torch < 2.0 versions with
+    `remove_duplicate` option added.
+
+    Args:
+        memo: a memo to store the set of modules already added to the result
+        prefix: a prefix that will be added to the name of the module
+        remove_duplicate: whether to remove the duplicated module instances in the result
+            or not
```

Comment:
To conform with our doc process
```suggestion
        memo (set of `torch.nn.Module`, *optional*):
            A memo to store the set of modules already added to the result
        prefix (`str`, *optional*): 
            A prefix that will be added to the name of the module
        remove_duplicate (`bool`, *optional*): 
            Whether to remove the duplicated module instances in the result or not
```